routes/confirmation/checkAnswers.js

```diff
@@ -0,0 +1,138 @@
+const answerInfo = [
+  { 
+    'sectionName': 'Personal Information',
+    'sectionLines': [
+      {
+        'text': 'Name',
+        'infoPath': ['personal.firstName', 'personal.lastName'],
+        'urlPath': '/personal/name',
+      },
+      {
+        'text': 'Date of Birth',
+        'infoPath': ['personal.dateOfBirth'],
+        'urlPath': '',
```

Comment:
Interesting. We could also not have a url at all if we don't need it and then the template could check for it. 

But not super bothered either way -- this works too.